<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_current_layer">
       <property name="text">
        <string>Current Layer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_current_layer">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_mean">
           <property name="text">
            <string>Mean</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="line_edit_mean">
           <property name="styleSheet">
            <string notr="true">.QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_median">
           <property name="text">
            <string>Median</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="line_edit_median">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_std_dev">
           <property name="text">
            <string>Std. Dev.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="line_edit_std_dev">
           <property name="styleSheet">
            <string notr="true">.QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_total">
           <property name="text">
            <string>Total</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="line_edit_total">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_data_point_count">
           <property name="text">
            <string>Count</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="line_edit_data_point_count">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_rms">
           <property name="text">
            <string>RMS</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="line_edit_rms">
           <property name="styleSheet">
            <string notr="true">.QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_snr">
           <property name="text">
            <string>SNR</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="line_edit_snr">
           <property name="styleSheet">
            <string notr="true">.QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Measured</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_measured_error">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">.QLabel {
	color: #a94442;
	background-color: #f2dede;
	padding: 10px;
	border: 1px solid #ebccd1;
	border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string>You must have at least three ROIs on the plot</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <property name="leftMargin">
             <number>12</number>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>12</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_equivalent_width">
              <property name="text">
               <string>Equivalent Width</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="line_edit_equivalent_width">
              <property name="styleSheet">
               <string notr="true">.QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_centroid">
              <property name="text">
               <string>Centroid</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="line_edit_centroid">
              <property name="styleSheet">
               <string notr="true">.QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_flux">
              <property name="text">
               <string>Flux</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="line_edit_flux">
              <property name="styleSheet">
               <string notr="true">.QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_mean_continuum">
              <property name="text">
               <string>Mean Continuum</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="line_edit_mean_continuum">
              <property name="styleSheet">
               <string notr="true">.QLineEdit {
	background: #DDDDDD; 
	border: 1px solid #cccccc;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
